client/my-sites/marketing/do-it-for-me/landing.tsx

```diff
@@ -27,55 +54,75 @@ export default function DoItForMeLandingPage(): JSX.Element {
 		<Card>
 			<FormattedHeader
 				brandFont
-				headerText="Your Site Built By Wordpress"
-				subHeaderText="You want the website of your dreams. Our experts can create it for you."
+				headerText={ translate( 'Your Site Built By WordPress' ) }
+				subHeaderText={ translate(
+					'You want the website of your dreams. Our experts can create it for you.'
+				) }
 				align="left"
 			/>
-			<img
-				src="https://wpcom.files.wordpress.com/2020/10/built-by-hero-image.png"
-				width="100%"
-				alt="WordPress logo"
-			/>
 
-			<CardHeading tagName="h5" size={ 24 }>
-				Our website building plans are perfect for:
-			</CardHeading>
+			<p>
+				{ translate(
+					'Need a professionally designed website for your business fast? Choose from our custom designed templates below, add your content and media, and in just 1-2 days your website will be ready to launch, for just US$500. The one-time fee includes a 5-page website, a 30-minute Quick Start Zoom orientation about your site it’s complete. '
+				) }
+			</p>
+			<p>
+				{ translate(
+					'You’ll need to have content for 5 pages (Home, About, Contact, and 2 other pages based on your selected template). You can add that content in the submission form.'
+				) }
+			</p>
 
 			<VerticalsGrid>
-				<div>
-					<CardHeading tagName="h1" size={ 21 }>
-						Online Stores
-					</CardHeading>
-					<p>
-						Turn your website into an online store with everything you need to sell your products,
-						ideas, or services. Easily accept one-time or recurring payments, reach your customers,
-						and design your site so it stands apart — and converts more.
-					</p>
-				</div>
-				<div>
-					<CardHeading tagName="h1" size={ 21 }>
-						Educational Websites
-					</CardHeading>
-					<p>
-						Whether you’re creating webinars, podcasts, online courses, or more, we can help you
-						create a single platform that helps you connect with your audience, build a following,
-						and earn an income from your content.
-					</p>
-				</div>
-				<div>
-					<CardHeading tagName="h1" size={ 21 }>
-						Professional Services
-					</CardHeading>
-					<p>
-						Launch and grow your business with a professional design that helps you stand out,
-						marketing tools to help you reach your customers, and the functionality you need to keep
-						growing.
-					</p>
-				</div>
+				<Vertical>
+					<VerticalHeading>
+						{ translate( 'Professional Services' ) } <br /> { translate( 'Local Business' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Local business illustration' )` *ES Score: 7*